integration_tests/utils.py

```diff
@@ -9,3 +11,12 @@ def get_repo_root_path():
     rust_vmm_ci_path = os.path.dirname(integration_tests_path)
 
     return os.path.dirname(rust_vmm_ci_path)
+
+
+def get_cmd_output(cmd):
+    """Returns stdout content of `cmd` command."""
+    cmd_out = subprocess.run(cmd, shell=True, check=True,
+                             stdout=subprocess.PIPE,
+                             stderr=subprocess.PIPE)
```

Comment:
This function doesn't use `stderr` for anything, so no need to pipe it

Reply:
Done.